### PR TITLE
1.2.2 NOS Feeds changed

### PR DESCRIPTION
**Pull request by cow-n-berg**
All urls have changed to nos.nl/feeds. SailJail compatibility added.

### CODE REVIEW

**Review by Fuzzillogic — COMMENTED** [review]
Note the tech URL is "feeds.nos.nl/nosnieuwstech", not "feeds.nos.nl/nosnieuwstech**nologie**"

**Review by SanderKlootwijk — CHANGES_REQUESTED** [review]
Thank you @cow-n-berg once again for your contribution! This Sailfish app sometimes gets forgotten by me. I had already updated the Ubuntu Touch NedNieuws app.

Thanks to @Fuzzillogic for reviewing the pull request. I also took a look at it. In the FirstPage.qml the URL for Tech news is indeed incorrect. It should be "http://feeds.nos.nl/nosnieuwstech". Furthermore, the CoverDelegate.qml still contains old Feedburner URLs. These URLs should change to the new feeds.nos.nl format too. If you would like to change this, I'll merge the request and put a new version of the app in the Jolla Store :)

File: harbour-nednieuws.desktop [hunk 1]
```diff
@@ -4,3 +4,8 @@ X-Nemo-Application-Type=silica-qt5
 Icon=harbour-nednieuws
 Exec=sailfish-qml harbour-nednieuws
 Name=NedNieuws
+
+[X-Sailjail]
+OrganizationName=org.sanderklootwijk
+ApplicationName=news-photos
```
Comment (a-dekker) [inline harbour-nednieuws.desktop:10]: ApplicationName is not correct

**Comment by a-dekker** [comment]
Please try to exclude those \*.pro.user.\* files in your commits/PR. They are dynamically generated and should not be part of the sourcecode.

**Pull request closed without merging**